express-api/src/constants/urls.ts

```diff
@@ -6,5 +6,9 @@ const urls = {
     AUTH: process.env.CHES_AUTH_URL,
     HOST: process.env.CHES_HOST_URI,
   },
+  LTSA: {
+    AUTHURL: 'https://appsuat.ltsa.ca/iam/api/auth/login/integrator',
+    HOSTURI: 'https://tduat-x42b.ltsa.ca/titledirect/search/api/',
```

Comment:
Should these be environment variables as well? There are different URLs for prod.